src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -1402,6 +1318,141 @@ type TableContext<'TRecord>
         else
             t.VerifyTableAsync()
 
+    member t.Transaction() =
+        match metricsCollector with
+        | Some metricsCollector -> Transaction(metricsCollector = metricsCollector)
```

Comment:
`Transaction(?metricsCollector = metricsCollector)`
also I would call this `CreateTransaction` or maybe `Transact` - verbing a noun is not very intuitive?